### PR TITLE
pod label changed to 'deployment' from 'deploymentconfig'

### PR DESCRIPTION
**Pull request by pruan-rht**

### CODE REVIEW

**Comment by xiuwang** [comment]
@pruan-rht  the label 'deploymentconfig' changeing to 'deployment'  is since 4.4+ , then your change won't work below 4.4.
Could you change to use other pod label as selector?

File: features/cli/pod.feature [hunk 1]
```diff
@@ -201,15 +201,15 @@ Feature: pods related scenarios
   Scenario: 4.0 Oauth provider info should be consumed in a pod
     Given I have a project
     When I run the :new_app client command with:
-      | docker_image     | aosqe/ruby-ex        |
+      | docker_image | aosqe/ruby-ex |
```
Comment (liangxia) [inline features/cli/pod.feature:204]: We can use `-l/--labels` when new-app.
```
-l, --labels='': Label to set in all resources for this application.
```

**Comment by pruan-rht** [comment]
@xiuwang @liangxia thanks for the suggestion...PR updated.  PTAL again. thx!

**Pull request merged**